yogstation/code/modules/admin/verbs/fix_air.dm

```diff
@@ -18,3 +18,41 @@
 			GM.parse_gas_string(F.initial_gas_mix)
 			F.copy_air(GM)
 			F.update_visuals()
+
+/client/proc/fix_air_z()
+	set name = "Fix Air on current Z-level"
+	set category = "Misc.Unused"
+	set desc = "Fixes air on the entire z-level, temporarily disables atmos"
+
+	if(!holder)
+		to_chat(src, "Only administrators may use this command.", confidential=TRUE)
+		return
+	if(!check_rights(R_ADMIN,1))
```

Comment:
```suggestion
	if(!check_rights(R_ADMIN, TRUE))
```

Reply:
The default value for the second argument is already a truthy value so I just removed it